packages/keystone/README.md

```diff
@@ -120,18 +120,18 @@ const keystone = new Keystone({
 
 ## Methods
 
-| Method                | Description                                                                  |
-| --------------------- | ---------------------------------------------------------------------------- |
-| `connect`             | Manually connect to Adapters.                                                |
-| `createAuthStrategy`  | Creates a new authentication middleware instance.                            |
-| `createItems`         | Add items to a `Keystone` list.                                              |
-| `createList`          | Add a list to the `Keystone` schema.                                         |
-| `disconnect`          | Disconnect from all adapters.                                                |
-| `executeQuery`        | Run GraphQL queries and mutations directly against a `Keystone` instance.    |
-| `extendGraphQLSchema` | Extend keystones generated schema with custom types, queries, and mutations. |
-| `prepare`             | Manually prepare `Keystone` middlewares.                                     |
-| `createContext`       | Create a `context` object that can be used with `executeGraphQL()`.          |
-| `executeGraphQL`      | Execute a server-side GraphQL operation within the given context.            |
+| Method                | Description                                                                            |
+| --------------------- | -------------------------------------------------------------------------------------- |
+| `connect`             | Manually connect to Adapters.                                                          |
+| `createAuthStrategy`  | Creates a new authentication middleware instance.                                      |
+| `createItems`         | Add items to a `Keystone` list.                                                        |
+| `createList`          | Add a list to the `Keystone` schema.                                                   |
+| `disconnect`          | Disconnect from all adapters.                                                          |
+| `executeQuery`        | (Deprecated) Run GraphQL queries and mutations directly against a `Keystone` instance. |
```

Comment:
I wish I knew which version it is deprecated? should I create a PR to add that here.
We have not used such versioning information in documents. seems like the need of hour

Reply:
I'm not sure I understand the need here?

Reply:
more info - https://github.com/keystonejs/keystone/discussions/3180